docs/content/modeling/getting-started.mdx

```diff
@@ -354,28 +336,24 @@ Now that we have a list of object types we can start defining them using the <Up
   }}
 />
 
-:::info Caution
-You're now in the process of building a version you can use. The model above is not yet a valid authorization model accepted by <ProductName format={ProductNameFormat.ShortForm}/>.
-:::
-
 :::info Important
-In a few cases other users can be part of determining whether an action can be performed on an action or not. Social media is an example of this "a user can comment on a picture if they are a friend of the user that published it".
+Though you're in the process of building a version you can use, the model above is not yet a valid authorization model accepted by <ProductName format={ProductNameFormat.ShortForm}/>.
+
+In a few cases, other users can determine whether one action can be performed on another action. For example, on social media, sometimes a user can comment on a picture if they are a friend of the user that published it.
 
-In those cases [**User** should also be an object type](./building-blocks/object-to-object-relationships.mdx). Following the last recommendation, we would discover the User type because it is a second noun in an expression: "friend of the user".
+In those cases [**User** must be an object type](./building-blocks/object-to-object-relationships.mdx), and you would discover the User type because it is a second noun in an expression `friend of the user`.
 :::
 
 ### 03. List Relations For Those Types
 
 ![List relations for those types](./assets/getting-started-diagram-04.svg)
 
-Each of the previously defined types has a set of relations. <ProductConcept section="what-is-a-relation" linkName="Relations" /> are an important component in your model. After all, <ProductName format={ProductNameFormat.ShortForm}/> is a <IntroductionSection linkName="Relationship Based Access Control (ReBAC)" section="what-is-relationship-based-access-control-rebac"/> system.
-
-To identify relations for a type we in the write-up we can perform an exercise similar to the one we did in [list the type of objects in your system](#02-list-the-object-types).
+Each previously defined type has a set of relations. A process similar to that above to [list the type of objects in your system](#02-list-the-object-types) defines your system's relations.
 
 Relations for a type \{type\} will be all of these:
 
-- any noun that is the \{noun\} of a "\{noun\} of a/an/the \{type\}" expression. **These are typically the Foreign Keys in a database.** We'll highlight these in <span className="green-highlight-text">green</span>.
-- any verb or action that is the \{action\} of a "can \{action\} (in) a/an \{type\}" expression. **These are typically the permissions for a type.** We'll highlight these in <span className="yellow-highlight-text">yellow</span>.
+- any noun that is the \{noun\} of a "\{noun\} of a/an/the \{type\}" expression. These are typically the Foreign Keys in a database, highlighted below in <span className="green-highlight-text">green</span>.
```

Comment:
I feel by removing the bold, we're no longer highlighting an important point we want to stress

Reply:
That's not how the Docs style guide recommends **bold** be used. One thing I'm going to do pretty soon is work on a modified style guide for FGA, but for now, I'd like to hew pretty closely to the original style guide.